.github/workflows/sync-dependencies.yml

```
@@ -0,0 +1,42 @@
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+      - name: Install Poetry
+        run: |
+          curl -sSL https://install.python-poetry.org | python3 -
+      - name: Extract template dependencies
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC1083:warning:11:4: This { is literal. Check expression (missing ;/\n?) or quote it

(shellcheck)